apps/mobile/src/features/Discover/useCases/SelectDappFromList/DAppTypes/DAppTypes.tsx

```
@@ -74,10 +74,10 @@ const TypeItem = ({
   const {palette: p} = useTheme()
 
   const [isPressed, setIsPressed] = React.useState(false)
-  const mappedStrings = useMappedStrings()
+  const strings = useStrings()
   const text = React.useMemo(
-    () => mappedStrings(name) ?? name,
-    [mappedStrings, name],
+    () => strings.useMappedStrings()(name) ?? name,
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)